packages.json

```diff
@@ -211,6 +211,7 @@
   "https://github.com/apple/swift-corelibs-xctest.git",
   "https://github.com/apple/swift-crypto.git",
   "https://github.com/apple/swift-distributed-tracing-baggage",
+  "https://github.com/apple/swift-distributed-tracing-baggage-core",
```

Comment:
I need to fix the nightly - this is what's been introducing the non-normalised urls I've flagged over in the validation PR.

Reply:
https://github.com/SwiftPackageIndex/PackageList-Validator/pull/7

Will close this PR for a re-run.